docs/source/examples/simple-landscape-pennylane.md

```diff
@@ -0,0 +1,237 @@
+---
+jupytext:
+  formats: md:myst,ipynb
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.23.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Mitigating the energy landscape of a variational circuit with Mitiq
+
+## This tutorial shows an example in which the energy landscape for a two-qubit variational circuit is explored with and without error mitigation.
+
+
+```{code-cell} ipython3
+import pennylane as qml
+from pennylane import numpy as np
+import matplotlib.pyplot as plt
+from mitiq.zne import mitigate_executor
+from mitiq.zne.inference import RichardsonFactory
+from mitiq.zne.scaling import fold_global as folding
+import pennylane_qiskit
+```
+
+## Defining the ideal variational circuit in Qiskit
+
+We define a function which returns a simple two-qubit variational circuit depending on a single parameter $\gamma$ (“gamma”).
+
+
+```{code-cell} ipython3
+# Describe noise
+noise_gate = qml.PhaseDamping
+noise_strength = 0.4
```

Comment:
```suggestion
noise_strength = 0.04
```

Reply:
To make it consistent with the Qiskit example, you should use a noise level of 0.04.

Reply:
For 0.4, I was getting almost the same plot as for the unmitigated circuit.